tox.ini

```diff
@@ -1,16 +1,14 @@
 [tox]
 envlist=
-    py{26,27,34}-defaultreactor
+    py{27,34}-defaultreactor
     py{35,36}-{default,qt5}reactor
     linting
 
 
 [testenv]
 deps=
     greenlet
-    py26: pytest<3.3
     py{27,34,35,36}: pytest
-    py26: twisted<15.5
     py{27,34,35,36}: twisted
```

Comment:
For both `pytest` and `twisted` dependencies, remove the `py{27,34,35,36}: `.  It's great I just learned how to make this work in time for py26 support to be removed.  :]

Reply:
:)  Done!